docs/content/examples/tutorials/basic-installation.md

```diff
@@ -84,142 +79,3 @@ services:
       - NET_ADMIN # For Fail2Ban to work
     restart: always
 ```
-
-## A Detailed Example
```

Comment:
This section has revisions that don't exist in a tagged release of docs.

Given review feedback, I think it's a bit premature to remove it. We could restore it here, or extract it out as a separate tutorial? The intent is defined early on into this section, so extracting it out should be fine?

In a future release when the docs are improved elsewhere, it can probably be simplified. For now it's quicker to keep around than to address all the feedback concerns in dropping it.

Reply:
👍🏼